tableauserverclient/server/endpoint/users_endpoint.py

```diff
@@ -70,15 +70,24 @@ def add(self, user_item):
     # Get workbooks for user
     @api(version="2.0")
     def populate_workbooks(self, user_item, req_options=None):
+        from .. import Pager
```

Comment:
Why do this here?

Reply:
I did that on the 'users-in-groups' bugfix and just copied the pattern.

You could say "Only imported when needed" but there's not really any penalty to just importing it at the top.

I can move it to a top-level import